WORKSPACE

```diff
@@ -104,6 +104,42 @@ http_archive(
     urls = ["https://github.com/WebAssembly/wabt/archive/39ceee53a9cd23451a28d4e56e151ef76044b5df.tar.gz"],
 )
 
+# Tool used for creating a compilation database.
+http_archive(
+    name = "io_kythe",
+    sha256 = "85dac12cdeea46f9e369ff109934aa98967bed1cd7c4c4afdc11577c3f99d31a",
+    strip_prefix = "kythe-4814f9f3fcc05c49fbe11f62f1e58a428048da27",
+    urls = [
+        # Head commit on 2019-12-03.
+        "https://github.com/kythe/kythe/archive/4814f9f3fcc05c49fbe11f62f1e58a428048da27.tar.gz",
+    ],
+)
+
+# Kythe tool dependency.
```

Comment:
Does `kythe_dependencies` work here, apart from pulling in extra deps? My understanding is that these deps are not actually downloaded unless they are actually needed anyways, or unless we run `bazel sync`, which we probably shouldn't do very often anyways. WDYT?

Reply:
`add_kythe_dependencies` contains `maybe` functions:
https://github.com/kythe/kythe/blob/25c5f5a88520250f9fd3605eb60a420605b1d458/setup.bzl#L30-L36
that are not executed without `bazel sync`, and the attempt to create a compilation database results in:
```
ERROR: error loading package '': in /root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/io_kythe/external.bzl: Unable to load file '@rule
s_java//java:repositories.bzl': file doesn't exist
```
So I explicitly added libraries that a necessary for CDB creation.